agenta-web/src/services/testsets/index.ts

```diff
@@ -0,0 +1,63 @@
+import useSWR from "swr"
```

Comment:
Not important, but we could also try to get rid of `useSWR` completely in favor of regular service functions (axios based), since this package is only used at 2 places in the app. Also, the possible inclusion of redux toolkit to the codebase will already take its place.